src/partials/advertisement.html

```diff
@@ -1,2 +1,7 @@
-<section class="advertisement">
-  </section>
\ No newline at end of file
+<section class="advertisement ">
+  <div class="container-discount">
+  <h2 class="visually-hidden">Advertisement</h2>
```

Comment:
add the word advertisement to all classes starting from visually hidden.
For example, <h2 class="advertisement-visually-hidden">Advertisement</h2>